myia/frontends/pytorch.py

```diff
@@ -116,6 +118,7 @@
     'abs': operations.array_abs,
     'argmax': argmax,
     'chunk': chunk,
+    'detach': detach,
```

Comment:
You can just set this directly to `operations.stop_gradient`

Reply:
Done!